hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/common/LinkFactoryImpl.java

```diff
@@ -62,39 +64,79 @@ private <T> String createNextLink(
         var sortOrders = pageable.getSort();
         var primarySort = Iterables.getFirst(sortOrders, null);
         var order = primarySort == null ? Direction.ASC : primarySort.getDirection();
-        var lastSort = Iterables.getLast(sortOrders, null);
-        // Only the last sort param's value is exclusive
-        var exclusiveParam = lastSort != null ? lastSort.getProperty() : null;
         var builder = UriComponentsBuilder.fromPath(request.getRequestURI());
         var paramsMap = request.getParameterMap();
         var paginationParamsMap = extractor.apply(lastItem);
+        var linkedMultiValueMap = new LinkedMultiValueMap<String, String>();
 
+        addParamMapToLink(paramsMap, paginationParamsMap, order, linkedMultiValueMap);
```

Comment:
the function name reads as add them to link, actually it adds items to a map. Adding to link is an operation which happens later.

```suggestion
        addParamMapToLink(paramsMap, paginationParamsMap, order, linkedMultiValueMap);
```

Reply:
Thanks I have updated these method names.